crates/tauri-plugin-holochain-service/android/src/main/java/com/plugin/holochain_service/HolochainServicePlugin.kt

```diff
@@ -56,6 +56,10 @@ class HolochainServicePlugin(private val activity: Activity): Plugin(activity) {
             "Holochain Service Running",
             NotificationManager.IMPORTANCE_HIGH
         ))
+        
+        // Attempt to connect to service
+        // It may not be running
+        this.serviceClient.connect()
```

Comment:
Unrelated, the service may be running already on app launch and we need to connect to it, so the UI can get the current status.